docs/source/getting-started/examples/accuracy.ipynb

```diff
@@ -0,0 +1,269 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Accuracy Method"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Many prior examples of queries used comparisons of the actual values to the computed differentially private values. Intuitively, we assessed the quality of our results by visually inspecting the proximity of the actual values to the differentially private outputs. \n",
+    "\n",
+    "However, this approach isn't concrete since interpretations can vary widely on scale, type of visualization, and other factors. A more objective way to evaluate differentially private outputs is by using the `accuracy` method. \n",
+    "\n",
+    "In this notebook ,we will: \n",
+    "- Explain the purpose and the application of the `accuracy` method. \n",
+    "- Clarify how to interpret the results of the `accuracy` method. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import opendp.prelude as dp\n",
+    "import polars as pl \n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np \n",
+    "\n",
+    "dp.enable_features(\"contrib\")\n",
+    "df = pl.scan_csv(\"sample_FR_LFS.csv\", infer_schema_length=1000, ignore_errors=True)\n",
+    "\n",
+    "context = dp.Context.compositor(\n",
+    "    data=df,\n",
+    "    privacy_unit=dp.unit_of(contributions=36),\n",
+    "    privacy_loss=dp.loss_of(epsilon=1.0),\n",
+    "    split_evenly_over=10,\n",
+    "    margins={\n",
+    "        (\"YEAR\", ): dp.Margin(public_info=\"keys\", max_partition_length=200_000, max_partition_contributions=4),\n",
+    "        (\"QUARTER\", ): dp.Margin(public_info=\"keys\", max_partition_length=200_000, max_partition_contributions=13),\n",
+    "        (\"YEAR\", \"QUARTER\",): dp.Margin(public_info=\"keys\", max_partition_length=200_000, max_partition_contributions=1),\n",
+    "        (): dp.Margin(max_partition_length=60_000_000),\n",
+    "    },\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We will use the same query we began with in [Part 1](Link Here): a count query for each year and quarter."
```

Comment:
CI won't pass with this. Possible courses:
- Get part 1 merged in so you can point at something
- Rephrase so it doesn't depend on anything outside
- change to http://example.com/TODO or similar -- but also move this to the hidden toctree